server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const app = express()
+const port = 5000
+
+app.use('/', express.static('./dist', {
```

Comment:
Wil je static files serveren vanuit de server? Ik denk dat dit niet echt meer nodig is, gezien de projecten zijn gescheiden.